src/solver/optimisation/opt_appel_solveur_lineaire.cpp

```diff
@@ -370,8 +382,14 @@ bool OPT_AppelDuSimplexe(const OptimizationOptions& options,
 
         Probleme.SetUseNamedProblems(true);
 
-        auto MPproblem = std::shared_ptr<MPSolver>(
-          ProblemSimplexeNommeConverter(options.ortoolsSolver, &Probleme).Convert());
+        auto ortoolsProblem = std::make_unique<OrtoolsLinearProblem>(Probleme.isMIP(), options.ortoolsSolver);
+        auto legacyOrtoolsFiller = std::make_unique<LegacyOrtoolsFiller>(ortoolsProblem->MPSolver(), &Probleme);
+        std::vector<LinearProblemFiller*> fillersCollection = {legacyOrtoolsFiller.get()};
+        LinearProblemData LP_Data;
+        LinearProblemBuilder linearProblemBuilder(fillersCollection);
+
+        linearProblemBuilder.build(*ortoolsProblem, LP_Data);
+        auto MPproblem = std::shared_ptr<MPSolver>(ortoolsProblem->MPSolver());
```

Comment:
This block appears twice in this file.
It is meant to create a **ortools MPSolver**.
It's a kind of code duplication.
This duplication will be removed in a further PR